installer/templates/new/test/views/error_view_test.exs

```diff
@@ -17,5 +17,18 @@ defmodule <%= application_module %>.ErrorViewTest do
   test "render any other" do
     assert render_to_string(<%= application_module %>.ErrorView, "505.html", []) ==
            "Server internal error"
+  end<% else %>test "renders 404.json" do
+    assert Poison.decode!(render_to_string(<%= application_module %>.ErrorView, "404.json", [])) ==
```

Comment:
In those cases, I would just call `render`, and not `render_to_string`, and compare the returned map with a map. :)